### ISSUE TITLE
Improvements in the components tabs

### ISSUE THREAD

**Issue by MuntashirAkon** [MuntashirAkon/AppManager#601]
There are a few rooms for improvements in the components tabs:
1. Long clicking on the **block/unblock** should allow a user to choose whether to block it or to only disable it (root only).
2. Batch/bulk operation in each tab (block/unblock, disable/enable, default).

**Comment by MuntashirAkon** [comment]
Of course, you can! Be sure to read the [contributing](https://github.com/MuntashirAkon/AppManager/blob/master/CONTRIBUTING.rst) section to understand how to contribute first.

Here's a task list for you (feel-free to input your own thoughts):

- Create a new context menu with three items: **IFW + disable**, **IFW**, **Disable** and **Default**. The context menu should appear when the user clicks on the **block/unblock** button in any of the four components tabs.
- Modify ComponentBlocker.java to support these four options (feel-free to ask me if you have trouble understanding how they work).
- RuleStorageManager should also store this new preference. To do that, you have to modify ComponentRule class to include an additional integer argument (assign 1-3 for the first three and 0 for the last item).
- Reuse the context menu for batch operations too. We have a custom component (based on Material BottomNavigation) for that called MultiSelectionView for this. But it isn't well documented right now. You can look into the RunningAppsActivity to understand how it works (or you can just post here if you are stuck).

Thanks.

**Comment by irenejoeunpark** [comment]
Hi @MuntashirAkon, I kept tried with different device settings, but could not get the sdk work on my mac. Sorry, but please assign this issue to someone else.

**Comment by MuntashirAkon** [comment]
> could not get the sdk work on my mac.

Try with a simple project first. App Manager is tested on macOS 11.6.1 (Intel CPU) where it builds without any issues. If you're using a silicon Mac, it could be an issue.

> Sorry, but please assign this issue to someone else.

OK. No problem.

**Comment by MuntashirAkon** [comment]
First three parts are implemented in d64f7ed8.
